[DOCS-1806] Document SCIM API operations that don't work for service accounts (#1926)
wandb/docs: platform/hosting/iam/scim.mdx

Retrieves information for a specific user in your organization.

> <Note>This operation does not retrieve team service accounts.</Note>

Author: ```suggestion
<Note>This operation does not retrieve service accounts.</Note>
```

> 

Reviewer: -team-. It doesn't retrieve any service accounts, org or team.

Retrieves a list of all users in your organization.

> <Note>This operation does not retrieve team service accounts.</Note>

Reviewer: It doesn't return org service accounts either. Maybe just say "This operation does not retrieve service accounts."

> <Note>This operation does not retrieve team service accounts.</Note>

Author: ```suggestion
<Note>This operation does not retrieve service accounts.</Note>
```

- **Dedicated Cloud** / **Self-Managed**: Sets the user's `active` field to `false`. To restore a deactivated user's access your organization, see [Reactivate user](#reactivate-user).
- **Multi-tenant Cloud**: Removes the user from the organization. To restore the user's access, re-add them to your organization. See [Create user](#create-user-request-multi-tenant). In Multi-tenant Cloud, a user's account is not managed by the organization.

> <Note>This operation works for users only, not service accounts. Delete a service account in the settings for the W&B Team.</Note>

Author: ```suggestion
<Note>This operation works for users only, not service accounts. Deactivate a service account in the settings for the W&B Team.</Note>
```

> <Note>This operation works for users only, not service accounts. Deactivate a service account in the settings for the W&B Team.</Note>

Reviewer: Service accounts can not be deactivated in the UI. We don't support SA deactivation.

> <Note>This operation works for users only, not service accounts. Deactivate a service account in the settings for the W&B Team.</Note>

Author: ```suggestion
<Note>This operation works for users only, not service accounts. Deactivating a service account is not supported. Manage team service accounts in the settings for the W&B Team.</Note>
```

Reactivates a previously deactivated user in your organization.

<Note>

> - User reactivation works for users only, not service accounts. Reactivate a service account in the settings for the W&B Team.

Reviewer: Same as above comment. We don't support deactivation/reactivation for service accounts.

> - User reactivation works for users only, not service accounts. Reactivate a service account in the settings for the W&B Team.

Author: ```suggestion
- User reactivation works for users only, not service accounts. Reactivation is not supported for service accounts. Manage service accounts in the settings for the W&B Team.
```

Assigns an organization-level role to a user.

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Reviewer: Service accounts don't support custom roles (yet).

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Author: ```suggestion
<Note>This operation works for users only, not service accounts. Custom roles are not supported for service accounts.</Note>
```

Assigns a team-level role to a user.

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Reviewer: Same as above.

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Author: ```suggestion
<Note>This operation works for users only, not service accounts. Custom roles are not supported for service accounts.</Note>
```

Adds a user to a registry with an assigned registry-level role.

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Reviewer: Same as above. Maybe we should just call out above that SCIM does not support editing roles for service accounts in general, rather than calling it out explicitly for every operation?

> <Note>This operation works for users only, not service accounts. Assign roles to a service account in the settings for the W&B Team.</Note>

Author: ```suggestion
<Note>This operation works for users only, not service accounts. Custom roles are not supported for service accounts.</Note>
```
